### ISSUE TITLE
Inconsistent Results in findStation() for small distances

### ISSUE THREAD

**Issue by johnpauls** [IMDProjects/IM_Climate#50]
I seem to be getting inconsistent results from `findStation()`. For example:

```
> findStation("CATO",distance=2)
    uid              name longitude latitude          sid1 sid1_type          sid2 sid2_type    sid3 sid3_type state elev    minDate
1  9388 CATOCTIN MTN PARK -77.48306 39.64528      181530 2      COOP USC00181530 6      COOP THUM2 7     NWSLI    MD 1610 1968-01-01
2 53782  THURMONT 0.8 SSE -77.40500 39.61250 US1MDFR0010 6      GHCN   MDFR0010 10  CoCoRaHS    <NA>      <NA>    MD  501 2008-09-06
3 62465        CAMP DAVID -77.46806 39.64500       03749 1      WBAN USW00003749 6      GHCN    <NA>      <NA>    MD 1847 2006-05-01
     maxDate unitCode
1 2013-11-13     CATO
2 2016-10-24     CATO
3 2013-08-07     CATO
```

With a `distance=2`, there are 3 station and 2 are inside the park. `getBBox` confirms this

```

getBBox("CATO",0)
[1] "-77.5048330165446, 39.6198606602848, -77.4176096394658, 39.6920480206609"

```

but if I use a smaller distance, then I get errors instead of the 2 CATO stations:

```
> findStation("CATO",distance=1)
Error in data.frame(..., check.names = FALSE) : 
  row names contain missing values
```

```
findStation("CATO",distance=0)
Error in data.frame(..., check.names = FALSE) : 
  row names contain missing values
```

**Comment by BrentFrakes** [comment]
Unable to replicate on the python side:

import StationFinder

sf = StationFinder.StationFinder()

stations = sf.findStation(unitCode = 'CATO', distance = 2)
print stations

stations = sf.findStation(unitCode = 'CATO', distance = 1)
print stations

stations = sf.findStation(unitCode = 'CATO', distance = 0)
print stations

**Comment by llnelson** [comment]
Fixed in changeset 7c94c9b (dev branch) - 20161118.
